docs/_sass/_sidebar.scss

```diff
@@ -7,19 +7,116 @@
   background: $color-lightgray;
   border-right: 1px solid darken($color-lightgray, 5%);
   overflow-y: auto;
-
-  @include respond-to('small') {
-    /* Todo: show navigation in off-site-menu with hamburger icon */
-    display: none;
-  }
+  outline: none;
 
   .site-name {
     margin-right: -1px;
     text-align: center;
     padding: 20px;
-    background: $color-primary;
     background: linear-gradient($color-primary, darken($color-primary, 8%));
     color: $title-color;
-    font-size: 20px;
+
+    a {
+      font-size: 3rem;
+      color: inherit;
+      margin: 6px;
+      display: block;
+      border-bottom: none;
+      h1 {
+        margin: 0;
+        font-size: inherit;
+      }
+    }
+
+    a:focus {
+      outline: none;
+    }
+
+    a:hover {
+      transition: .2s;
+      text-shadow: 0 0 20px white;
+    }
+
+    .github-button {
+      font-size: 1rem;
+    }
+
+    iframe {
+      margin: 6px;
+    }
+  }
+
+  .menu {
+    float: left;
+    display: none;
+    padding: 10px;
+    border: 1px solid white;
+    border-radius: 3px;
+    cursor: pointer;
+  }
+
+}
+
+@include respond-to('medium') {
+  .sidebar {
+    display: block;
+    position: fixed;
+    width: 100%;
+    height: 50px;
+    overflow: visible;
+
+    .site-name {
+      .github-button, iframe {
+        float: right;
+      }
+
+      width: 100%;
+      height: 50px;
+      padding: 5px;
+
+      a {
+        font-size: 1.5rem;
+        display: inline-block;
+      }
+    }
+
+    li {
+      margin: 5px;
+      padding: 10px;
+    }
+
+    .menu.hide {
+      display: none;
+    }
+
+    .menu.show {
+      display: inline;
+    }
+
+    .navigation {
+      transition: .5s .1s height, .1s .3s opacity;
+      height: 0;
+      overflow: auto;
+      opacity: 0;
+      background: $color-lightgray;
+      scroll-behavior: smooth;
```

Comment:
It thinks it found incorrect CSS key. But it just stupid :)
https://developer.mozilla.org/en/docs/Web/CSS/scroll-behavior